terra_notebook_utils/drs.py

```diff
@@ -302,10 +306,22 @@ def copy_to_bucket(drs_uri: str,
     },
 }
 
-def copy_batch(manifest: List[Dict[str, str]],
-               indicator_type: Indicator=Indicator.notebook_bar if is_notebook() else Indicator.log,
-               workspace_name: Optional[str]=WORKSPACE_NAME,
-               workspace_namespace: Optional[str]=WORKSPACE_NAMESPACE):
+def copy_batch(drs_urls: Iterable[str],
+               dst_pfx: str,
+               indicator_type: Indicator = Indicator.notebook_bar if is_notebook() else Indicator.log,
+               workspace_name: Optional[str] = WORKSPACE_NAME,
+               workspace_namespace: Optional[str] = WORKSPACE_GOOGLE_PROJECT):
```

Comment:
This:  
`workspace_namespace: Optional[str] = WORKSPACE_GOOGLE_PROJECT):`
must be changed to:  
`workspace_namespace: Optional[str]=WORKSPACE_NAMESPACE):`

Reply:
Whoops, that must have slipped through when I rebased. Nice catch.